onnx/mapping.py

```diff
@@ -66,5 +66,9 @@
     int(SequenceProto.TENSOR): 'tensor_values',
     int(SequenceProto.SPARSE_TENSOR): 'sparse_tensor_values',
     int(SequenceProto.SEQUENCE): 'sequence_values',
-    int(SequenceProto.MAP): 'map_values'
+    int(SequenceProto.MAP): 'map_values',
+    int(OptionalProto.TENSOR): 'tensor_value',
```

Comment:
I think this is not going to work, because SequenceProto.TENSOR and OptionalProto.TENSOR have same (key) value. We will have to use two different maps, one for Sequence, and one for Optional.